python/ray/tests/test_memstat.py

```diff
@@ -37,6 +37,8 @@
 OBJECT_SIZE = "object size"
 REFERENCE_TYPE = "reference type"
 
+avoid_multi_node = (os.name == "nt")
```

Comment:
Is this variable for better readability? I am interesting in knowing why we this check is not directly written in L238 below?

Reply:
The idea is that someday we can relax this restriction. Once that happens, we can search-and-replace all the "avoid_multi_node" occurrences.